pkgs/applications/misc/1password-gui/default-electron.nix

```
@@ -0,0 +1,67 @@
+{ lib, stdenv
+, fetchurl
+, appimageTools
+, makeWrapper
+, electron_13
```

```suggestion
, electron
```
please do ``electron = electron_13`` in the top-level call.

Where is this file actually used?